### ISSUE TITLE
Strace does not work on toolbox

### ISSUE THREAD

**Issue by Reifier** [coreos/bugs#1734]
# Issue Report #
### CoreOS Version ###
$ uname -r
4.7.3-coreos-r3

```
$ cat /etc/os-release
DISTRIB_ID=CoreOS
DISTRIB_RELEASE=1185.5.0
DISTRIB_CODENAME="MoreOS"
DISTRIB_DESCRIPTION="CoreOS 1185.5.0 (MoreOS)"
ID=ami
VERSION_ID=0.0.7
NAME="Amazon EC2"
HOME_URL="http://aws.amazon.com/ec2/"
BUG_REPORT_URL="https://github.com/coreos/bugs/issues"
NAME=CoreOS
ID=coreos
VERSION=1185.5.0
VERSION_ID=1185.5.0
BUILD_ID=2016-12-07-0937
PRETTY_NAME="CoreOS 1185.5.0 (MoreOS)"
ANSI_COLOR="1;32"
HOME_URL="https://coreos.com/"
BUG_REPORT_URL="https://github.com/coreos/bugs/issues"
```

### Environment ###
Dev
What hardware/cloud provider/hypervisor is being used to run CoreOS?
aws 
### Expected Behavior ###
strace -s 99 -ffp ${pid} 
should give the trace. 
### Actual Behavior ###
Instead it gives:
strace: attach: ptrace(PTRACE_SEIZE, 17102): Operation not permitted
### Reproduction Steps ###

  1. start toolbox
  2. run strace on a process of your choice

### Environment ###

What hardware/cloud provider/hypervisor is being used to run CoreOS?

**Comment by euank** [comment]
strace is installed in the base OS as well. Is there a reason why it should be run in the toolbox? If a different version of strace is desired, running it using rkt with [stage1-fly](https://coreos.com/rkt/docs/latest/running-fly-stage1.html) or docker with `--privileged --pid=host` should work too.

Regardless, the error in the toolbox is because `systemd-nspawn` applies a seccomp filter. I'm not sure if it can be trivially disabled. cc @lucab, happen to know?
This would also be fixed by #1610.

**Comment by Reifier** [comment]
No, there is no reason for this to be in a toolbox container. I did not realize that strace is available on the general system since the loud voice of propaganda was kind of: "All your debugging tools in a toolbox!".

Thanks for pointing that out.
